src/args.rs

```diff
@@ -731,8 +731,10 @@ impl<'a> ArgMatches<'a> {
         let mut specs = vec![
             "path:fg:magenta".parse().unwrap(),
             "line:fg:green".parse().unwrap(),
+            "column:fg:yellow".parse().unwrap(),
```

Comment:
I think I'd prefer if you backed out the addition of `column` and `separator` color styles. Whether we add that or not should really be decided in #377.